tox.ini

```diff
@@ -1,16 +1,52 @@
 [tox]
-envlist = py27, py33, py34, py35, pypy, jython, pylint
+envlist = coverage-erase, py27, py33, py34, py35, pypy, jython, pylint, coveralls
 skip_missing_interpreters = true
 
 [testenv:pylint]
+setenv =
+    COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 deps =
    git+https://github.com/PyCQA/astroid@master
+   coverage
 
-commands = pylint -rn --rcfile={toxinidir}/pylintrc {envsitepackagesdir}/pylint
```

Comment:
What is this line trying to accomplish? Running coverage with pylint seems useless.